test/server/cards/01_SOR/units/SeventhSisterImplacableInquisitor.spec.ts

```diff
@@ -0,0 +1,62 @@
+describe('Seventh Sister', function () {
+    integration(function (contextRef) {
+        describe('Rukh\'s triggered ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['battlefield-marine', 'seventh-sister#implacable-inquisitor'],
+                        hand: ['heroic-resolve'],
+                    },
+                    player2: {
+                        groundArena: ['wampa', 'pyke-sentinel'],
+                        spaceArena: ['cartel-spacer'],
+                        leader: { card: 'boba-fett#daimyo', deployed: true },
+                        base: 'chopper-base'
+                    },
+                });
+            });
+
+            it('may deal 3 damage to a ground unit if attacked base', function () {
+                const { context } = contextRef;
+
+                const reset = (passAction = true) => {
+                    context.seventhSisterImplacableInquisitor.exhausted = false;
+                    context.seventhSisterImplacableInquisitor.damage = 0;
+                    context.wampa.damage = 0;
+                    context.chopperBase.damage = 0;
+                    if (passAction) {
+                        context.player2.passAction();
+                    }
+                };
+
+                context.player1.clickCard(context.seventhSisterImplacableInquisitor);
+                expect(context.player1).toBeAbleToSelectExactly([context.wampa, context.pykeSentinel, context.bobaFettDaimyo, context.chopperBase]);
+                context.player1.clickCard(context.chopperBase);
+                expect(context.player1).toBeAbleToSelectExactly([context.wampa, context.pykeSentinel, context.bobaFettDaimyo]);
+                context.player1.clickCard(context.wampa);
+                expect(context.wampa.damage).toBe(3);
+
+                // CASE 2: may not deal 3 damage if attacked unit
+                reset();
+                context.player1.clickCard(context.seventhSisterImplacableInquisitor);
+                expect(context.player1).toBeAbleToSelectExactly([context.wampa, context.pykeSentinel, context.bobaFettDaimyo, context.chopperBase]);
+                context.player1.clickCard(context.wampa);
+                expect(context.wampa.damage).toBe(3);
+                expect(context.player2).toBeActivePlayer();
+
+                // CASE 3: should trigger damage if damage to base is dealt by overwhelm
+                reset();
+
+                context.player1.clickCard(context.heroicResolve);
```

Comment:
If the test requires different setup (i.e. starting with an upgrade) in that case I'm fine creating a different describe block, I think it's clearer. You can start her in play with the upgrade attached with `groundArena: [{ card: 'seventh-sister#implacable-inquisitor', upgrades: ['heroic-resolve'] }`